src/InfoViz/Native/FieldSelector/index.js

```diff
@@ -71,8 +72,8 @@ function fieldSelector(publicAPI, model) {
     function renderField(d, i) {
       const field = model.provider.getField(d);
       const fieldContainer = d3.select(this);
-      let legendCell = fieldContainer.select('.legend');
-      let fieldCell = fieldContainer.select('.field');
+      let legendCell = fieldContainer.select(`.${style.legend}`);
+      let fieldCell = fieldContainer.select(`.${style.fieldName}`);
```

Comment:
You should not use style for making JavaScript selection. Instead use a class like jsLegendContainer.